### PR TITLE
[RU] umbrella-projects.md translation, and GLOSSARY.md update

### PR DESCRIPTION
**Pull request by yuriibodarev**
Hello, please check my translation and leave Your comments and suggestions. What do You think about  translation of `umbrella project` as `зонтичный проект`?

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @yuriibodarev! 

@brain-geek and @nscyclone, could you take a peek at this?

File: ru/lessons/advanced/umbrella-projects.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
+---
+layout: page
+title: Зонтичные проекты
+category: advanced
+order: 8
+lang: ru
+---
+
+Со временем ваш проект может разрастись до больших размеров, на самом деле, даже до очень больших. В таком случае, инструмент `Mix` позволяет нам разделить код проекта на несколько приложений и сделать наши проекты на `Elixir` более управляемыми, по мере их роста.
```
Comment (nscyclone) [inline ru/lessons/advanced/umbrella-projects.md]: The comma before `по мере их роста` is not needed.

File: ru/lessons/advanced/umbrella-projects.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+{% include toc.html %}
+
+## Введение
+
+Зонтичный проект создаётся так же, как и любой обычный проект, при помощи `Mix`. Необходимо только передать `--umbrella` в качестве дополнительного параметра для `Mix`. В следующем примере мы создадим *оболочку* для системы машинного обучения. Почему именно система машинного обучения? А почему бы и нет? Такой проект не редко состоит из целого набора различных алгоритмов обучения и вспомогательных инструментов.
```
Comment (nscyclone) [inline ru/lessons/advanced/umbrella-projects.md]: `не редко` → `нередко`

File: ru/lessons/advanced/umbrella-projects.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    mix new my_app
+
+Commands like "mix compile" and "mix test" when executed
+in the umbrella project root will automatically run
+for each application in the apps/ directory.
+```
+
+Как вы можете увидеть из результатов выполнения команды в консоли, `Mix` создал для нас небольшой проект-заготовку с двумя директориями:
+
+  - `apps/` &mdash; тут будут находится наши под-проекты (дочерние проекты)
```
Comment (nscyclone) [inline ru/lessons/advanced/umbrella-projects.md]: `находится` → `находиться`
`под-проекты` → `подпроекты`
I think that `(дочерние проекты)` can be safely removed, the meaning is clear enough without it.

File: ru/lessons/advanced/umbrella-projects.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+  - `config/` &mdash; тут будет распологаться конфигурация наших зонтичных проектов
```
Comment (nscyclone) [inline ru/lessons/advanced/umbrella-projects.md]: `распологаться` → `распологаться`

File: ru/lessons/advanced/umbrella-projects.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+│       ├── mix.exs
+│       └── test
+│           ├── test_helper.exs
+│           └── utilities_test.exs
+├── config
+│   └── config.exs
+└── mix.exs
+```
+
+Если мы перейдём обратно, к корню зонтичного проекта, то окажется, что мы можем использовать все обычные команды `Mix`, такие как компиляция. Так как все под-проекты это обычные приложения, вы можете перейти в их директории, и выполнять все те действия, которые вам обычно позволяет `Mix`.
```
Comment (nscyclone) [inline ru/lessons/advanced/umbrella-projects.md]: I'd replace `такие как` with `например` surrounded by commas.
`под-проекты` → `подпроекты`

File: ru/lessons/advanced/umbrella-projects.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+Consolidated Collectable
+Consolidated String.Chars
+Consolidated Enumerable
+Consolidated IEx.Info
+Consolidated Inspect
+```
+
+## IEx
+
+Вы можете подумать, что взаимодействие с приложениями в зонтичном проекте будет немного отличаться? На самом деле это не так! Если мы перейдём в корневую директорию проекта и запустим `IEx` командой `iex -S mix` &mdash; мы сможем нормально взаимодействовать со всеми проектами. Давайте, для примера, изменим код приложения `apps/datasets/lib/datasets.ex`:
```
Comment (nscyclone) [inline ru/lessons/advanced/umbrella-projects.md]: I suppose that there should be no question mark at the end of the first sentence.

**Comment by nscyclone** [comment]
Thanks, @yuriibodarev!
I've just read it and left some notes for you. I am also looking for any references of the Russian translation for `umbrella projects` but didn't find anything worthwhile yet.

**Comment by yuriibodarev** [comment]
@nscyclone fixed errors according to Your comments and updated translation

File: ru/lessons/advanced/umbrella-projects.md [hunk 7]
```diff
@@ -0,0 +1,212 @@
+---
+layout: page
+title: Зонтичные проекты
+category: advanced
+order: 8
+lang: ru
+---
+
+Со временем ваш проект может разрастись до больших размеров, на самом деле, даже до очень больших. В таком случае, инструмент `Mix` позволяет нам разделить код проекта на несколько приложений и сделать ваши проекты на `Elixir` более управляемыми по мере их роста.
```
Comment (brain-geek) [inline ru/lessons/advanced/umbrella-projects.md]: `размеров, на самом деле` -> `размеров. На самом деле` ?

Comment (brain-geek) [inline ru/lessons/advanced/umbrella-projects.md]: And in the second part - it's mixture of 'ours' and 'yours'

File: ru/lessons/advanced/umbrella-projects.md [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+
+{% include toc.html %}
+
+## Введение
+
+Зонтичный проект создаётся так же, как и любой обычный проект, при помощи `Mix`. Необходимо только передать `--umbrella` в качестве дополнительного параметра для `Mix`. В следующем примере мы создадим *оболочку* для системы машинного обучения. Почему именно система машинного обучения? А почему бы и нет? Такой проект нередко состоит из целого набора различных алгоритмов обучения и вспомогательных инструментов.
```
Comment (brain-geek) [inline ru/lessons/advanced/umbrella-projects.md]: Obsolete `только`?

Comment (brain-geek) [inline ru/lessons/advanced/umbrella-projects.md]: what's the `оболочку` term? Do we have it in glossary? :)

**Comment by brain-geek** [comment]
Hey, @yuriibodarev , I've left some feedback also.

File: ru/lessons/advanced/umbrella-projects.md (hunk 8 above)
Comment (yuriibodarev) [inline ru/lessons/advanced/umbrella-projects.md]: `оболочку` is the translation of the `shell` word from the text and it's a description of an example and not the `Elixir` term

**Comment by yuriibodarev** [comment]
@brain-geek translation updated

**Comment by brain-geek** [comment]
@yuriibodarev Everything is good with texts, but can you squash the commits?

**Comment by yuriibodarev** [comment]
@doomspork  done

**Comment by brain-geek** [comment]
Thanks, @yuriibodarev !

**Pull request merged**